cmd/lsjson/lsjson.go

```diff
@@ -75,7 +75,7 @@ If ` + "`--no-mimetype`" + ` is specified then MimeType will be blank. This can
 speed things up on remotes where reading the MimeType takes an extra
 request (e.g. s3, swift).
 
-If ` + "`--encrypted`" + ` is not specified the Encrypted won't be emitted.
+If ` + "`--encrypted`" + ` is not specified the Encrypted will be omitted.
```

Comment:
> the Encrypted will

That sounds a bit off still, should insert the word "property" here as well I think?
Nothing with your change, which is an improvement!
What about extending with a little bit more detail as well:

...`is not specified, or the remote is not encrypted, the Encrypted and EncryptedPath properties`...